### PR TITLE
Master fixes

### PR DESCRIPTION
**Pull request by alexcouper**
Fixes following code review in #1
- [x] [Change to declaring upstream dependencies](https://github.com/ployst/component-builder/pull/1#discussion_r84461487)
- [x] Run things from within their project dir.

### CODE REVIEW

File: src/component_builder/config.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import os.path
-from ConfigParser import ConfigParser
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
```
Comment (alexcouper) [inline src/component_builder/config.py:5]: Re: python2 vs python3. I wonder what the best way would be to ensure this in tests.

Probably `make test` _does_ need to care about virtualenvs after all.

Comment (txels) [reply src/component_builder/config.py:5]: `tox`?

**Review by txels — APPROVED** [review]
MoG

**Comment by txels** [comment]
Ah, `make build .` in circle.yml no longer works now.

**Pull request merged**